src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -342,4 +352,25 @@ public boolean containsQualifier(@NonNull String jsonFileName) {
     public void removeQualifier(@NonNull String jsonFileName) {
         qualifiers.remove(jsonFileName);
     }
+
+    /**
+     * Exclude the specified {@link ChartType} from being rendered.
+     *
+     * @param chartTypes charts not to be rendered
+     */
+    public void chartsNotToRender(@NonNull ChartType... chartTypes) {
```

Comment:
How about this convention https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/Configuration.java#L305 ?

Reply:
Regarding convention, are you looking for the `method name` to be more like `setChartsNotToRender()`? Or to change param from `varargs` to `Set`? Or both? :-)

Reply:
I thought about Set but yes, name also could be improved or even should :-)